packages/auth/openapi/id-provider.yaml

```diff
@@ -0,0 +1,162 @@
+openapi: 3.1.0
```

Comment:
Should OpenAPI spec validation be added to GitHub Actions, a la:
https://github.com/interledger/open-payments/blob/1e3e118d8b22c5d2942f972e28ebf0f0114d04f5/.github/workflows/validate-openapi.yaml#L26-L32

Seems like it would go here (and the check would need to run `fetch-schemas`?):
https://github.com/interledger/rafiki/blob/1b03f3e766735ec7c9618bc044838364723ad4f2/.github/workflows/lint_test_build.yml#L41
I'm assuming where already covered with formatting
https://github.com/interledger/rafiki/blob/1b03f3e766735ec7c9618bc044838364723ad4f2/.github/workflows/lint_test_build.yml#L19

Reply:
Good idea, have added #724